src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/hybrid_2018_03_01/test_vm_defaults.py

```diff
@@ -340,50 +340,6 @@ class TestBigVMSSDefaults(unittest.TestCase):
     def _set_up_ns(cls, ns):
         ns.single_placement_group, ns.zones, ns.platform_fault_domain_count, ns.instance_count = None, None, None, 2
 
-    def test_vmss_disable_single_placement_group(self):
```

Comment:
Why are we removing these tests?  Single placement group is still a feature, so we should still keep the tests in place. The only thing we would not need to test is the validation you removed.

Reply:
These tests were testing the client side validation I removed. 

I could try to add tests for the parts remaining in the validator